adr/0009-add-faker-for-wanda.md

```diff
@@ -0,0 +1,24 @@
+# 9. Add Faker for Wanda
+
+Date: 2023-02-09
+
+## Status
+
+Accepted
+
+## Context
+
+A demo environment is needed to showcase the functionality of the execution server. The current implementation uses live data, which is not ideal for demonstration purposes. A Faker-based implementation will provide a safe and controlled environment for demos.
+
+## Decision
+
+1.  Create a new Elixir module that implements the ServerBehaviour using the Faker library.
```

Comment:
Here the decision shouldn't talk about "code" things. It should talk that we chose to fake the demo evaluations in wanda, instead of other places.

Reply:
Since I've discussed this in the context, I think the decision part should document a bit the steps taken to implement this. Do you think it's still needed to add anything else here?

Reply:
Actually, I would put the bullet list from the `Context` chapter in the `Decision` and remove this. I still don't like having so low level technical details in a ADR.